friends/src/components/FriendForm.js

```diff
@@ -0,0 +1,58 @@
+import React, { useState } from "react";
+
+const FriendForm = props => {
+
+const [friend, setFriend] = useState({
+    name: "",
+    age: "",
+    email: "",
+});
+
+const handleChange = e => {
+    setFriend({...friend, [e.target.name]: e.target.value});
+};
+
+const handleSubmit = e => {
+    e.preventDefault();
+    props.addFriend(friend);
+    setFriend({
+        name: "",
+        age: "",
+        email: "",
+    });
```

Comment:
After adding friends where you able to add in the handleSubmit `props.history.push(`/YourFriendsList')`